localplanning.services/src/lib/lpa-api/index.ts

```diff
@@ -22,13 +22,29 @@ export interface LPA {
   domain: string | null;
 }
 
-const EXCLUDED_TEAM_SLUGS = [
-  "open-digital-planning",
-  "opensystemslab",
-  "planx",
-  "templates",
-  "testing",
-  "wikihouse",
+const TEAMS_ALLOW_LIST = [
```

Comment:
In a previous PR, this was an allow list of teams which is much safer - this must have been lost between branches somewhere.